2/1_peer_info.md

```diff
@@ -1,8 +1,32 @@
-* Peer info, what is it, what does it store?  Documentation for the API: (https://github.com/libp2p/js-peer-info/tree/v0.15.1)
+Peer Identity
+=============
+
+One of the key concepts of libp2p is how the identity of a peer is established.  We talked about multiaddresses in the previous section which tell libp2p how to connect to a specific node and the critical data element that uniquely identifies a node is it's [`PeerId`](https://docs.libp2p.io/concepts/peer-id/).  Each libp2p has a secret, private key along with a corresponding public key that can be shared with other peers on the network.  This key pair is what enables libp2p nodes to communicate securely. The PeerId is a cryptographic hash of the node's public key.  There are several key types that can be used and the details are found in the [PeerId spec](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md).
+
+When setting up a libp2p node, you provide an `options` object to the `libp2p.create` function that tells libp2p what modules you are planning to use and can also provide additional configuration details as well.  Importantly, you can tell libp2p what your node's PeerId is as well as specify what addresses you want your node to listen on with code like below:
+```
+const node = libp2p.create({
+    addresses: {
+        listen: ['/ip4/127.0.0.1/tcp/0']
+    },
+    peerId: {
+        "id": "Qma3GsJmB47xYuyahPZPSadh1avvxfyYQwk8R3UnFrQ6aP",
+        "privKey": "CAASpwkwgg...",
+        "pubKey": "CAASpgIwggEiMA0G..."
+    }   
+})
+```
```

Comment:
Not sure if we need to mention this again as in the first chapter we explain about setting up the options object

Reply:
Hmm, I see what you're saying though since section 1 isn't really filled out, I don't have a point of reference for what's specified in the options map description in chapter 1.  Can we leave it for now and then come back to it once we merge the transport pr if needed?  I can open a new issue to remind us.